src/components/Rating.jsx

```diff
@@ -177,28 +214,29 @@ export default class Rating extends Component {
   }
 
   /**
-   * On focus callback will enable focus styling on component when focused using tab.
+   * enable focus styling of component when tab is used to focus component
    */
   _onFocus() {
     if(!this.active) {
       this.setState({focused: true});
     }
-    this.forceUpdate();
     if (this.props.onFocus) {
       this.props.onFocus(event);
     }
   }
 
   /**
-   * When user clicks the component this callback will update component value to that selected by user.
+   * update component when clicked
    */
   _onClick(event) {
     if(!this.props.disabled) {
       this._updateComponent();
     }
+
     if (this.props.onClick) {
       this.props.onClick(event);
     }
+    React.findDOMNode(this.refs.wrapper).blur();
```

Comment:
Actually the intent of putting blur there was not that focus style is not applied during click but rather that as click ends focus also end. But I guess probably mouseUp will be better place for this.

Ideally if focus comes with mouseDown it should go away with mouseUp.